compendium/DeclarativeServices/src/manager/ComponentConfigurationImpl.cpp

```diff
@@ -234,7 +234,9 @@ namespace cppmicroservices
                         {
                             for (auto const& config : configs)
                             {
-                                configNotifier->CreateFactoryComponent(config->GetPid(), mgr);
+                                auto properties = config->GetProperties();
+                                auto componentFactory = configNotifier->GetComponentFactory();
+                                componentFactory->CreateFactoryComponent(config->GetPid(), mgr, properties );
```

Comment:
Like "config->GetPid()", for consistency, "properties" should be passed as "config->GetProperties()" instead of declaring a local variable.